common/InternalRequestsCleaner.js

```diff
@@ -0,0 +1,21 @@
+const RequestsCleaner = require('./RequestsCleaner');
+
+class InternalRequestsCleaner extends RequestsCleaner {
+	verifySignatures() {
+		if (!this.calls) {
+			return;
+		}
+		Object.keys(this.calls).forEach((socketId) => {
```

Comment:
Beyond the scope of this PR as it's a significant decision to add `babel-polyfill` to the project